test/unit/test_default_inference_handler.py

```diff
@@ -178,33 +177,3 @@ def test_default_output_fn_gpu(inference_handler):
     output = inference_handler.default_output_fn(tensor_gpu, content_types.CSV)
 
     assert "1,2,3\n4,5,6\n".encode("utf-8") == output
-
-
-def test_eia_default_model_fn(eia_inference_handler):
```

Comment:
Why do we need to remove those?

We still have EIA specific logic.

Reply:
Seems to have been removed by mistake. Have been added back